kedro-telemetry/kedro_telemetry/plugin.py

```diff
@@ -90,17 +122,17 @@ def before_command_run(
             main_command = masked_command_args[0] if masked_command_args else "kedro"
```

Comment:
I can't comment on the line above since it hasn't been changed but could line 117 be changed to -
`cli = KedroCLI(project_path=metadata.project_path)` 
Following the changes in https://github.com/kedro-org/kedro/pull/3683, I think this will allow us to capture commands called from within subdirectories also and mask them properly